src/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css

```diff
@@ -805,7 +805,12 @@ html[data-theme="dark"] .highlight pre {
 }
 
 #version_switcher_menu a.list-group-item.active {
-  background-color: var(--pst-color-background);
-  border-color: var(--pst-color-border);
-  color: var(--pst-color-text-base)
+  background-color: #D09735!important;
```

Comment:
Do we really want to use this color, see https://www.color-hex.com/color/d09735.

I am not sure if this is in conflict with our style guidelines. I think it works fine for the dark theme, but for the light one it may not be the best.

Reply:
No, it is not conflicting IMO, since the navigation bar is always black (in dark and in light) and it is only at the time of hovering, if you are putting the dropdown. But i am open to change.